ocean_lib/data_provider/data_service_provider.py

```diff
@@ -350,6 +351,13 @@ def get_url(config):
     def get_api_version():
         return DataServiceProvider._remove_slash(os.getenv(ENV_PROVIDER_API_VERSION, DataServiceProvider.API_VERSION))
 
+    @staticmethod
+    def get_provider_information():
+        if DataServiceProvider.provider_info is None:
+            DataServiceProvider.provider_info = requests.get('http://localhost:8030')
```

Comment:
why is this using `localhost`?

Reply:
Shoul I get the url from config? I put localhost, because I wanted to be sure that I have the same URL from the browser, too.

Reply:
so everyone using ocean.py will have to run their own provider locally!